app/Console/Commands/RunBackupCommand.php

```
@@ -16,26 +18,36 @@ class RunBackupCommand extends Command
                     if ((new CronExpression((string) $backup->interval))->isDue(now(), config('app.timezone'))) {
-                        app(RunBackup::class)->run($backup);
-                        $total++;
+                        try {
+                            app(RunBackup::class)->run($backup);
+                            $total++;
+                        } catch (Throwable $e) {
+                            Log::warning('Failed to run backup', [
+                                'backup_id' => $backup->id,
+                                'server_id' => $backup->server_id,
+                                'error' => $e->getMessage(),
+                            ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'handle'. (undefined)

(StaticAccess)